src/server/api/controllers/questions.controller.js

```diff
@@ -0,0 +1,20 @@
+const knex = require('../../config/db');
+const Error = require('../lib/utils/http-error');
+
+const getQuestionById = async (id) => {
+  try {
+    const questions = await knex('questions')
+      .select('questions.id as id', 'latitude', 'longitude')
```

Comment:
By reading the heading of your PR i was expecting only latitude and longitude but you are retrieving id along with latitude and longitude